packages/kotti-ui/source/kotti-field-currency/types.ts

```diff
@@ -15,6 +15,7 @@ export namespace KottiFieldCurrency {
 		)
 		.extend({
 			currency: z.string(),
+			dataTest: z.string().optional(),
```

Comment:
This should be implemented globally, not per field.

Reply:
You are right, now is added into `potentiallySupportedPropsSchema`.